wfl/fit/mace.py

```diff
@@ -0,0 +1,205 @@
+		skip_if_present=True, run_dir=".", verbose=True, do_fit=True, remote_info=None,
+        remote_label=None, wait_for_results=True, **kwargs):
+    """
+        Fit MACE model.
+    Parameters
+    ----------
+    fitting_configs: ConfigSet
+       set of configurations to fit 
+    mace_fit_params: str or dict
+        parameters for fitting the model, it can be directly read from YAML file or passed on as dict. 
+    mace_fit_cmd: str
+        command for excecuting the MACE fitting. (For example, "python ~/path_to_mace_cripts/run_train.py")
+    mace_name: str
+        name of MACE label
+    run_dir: str, default '.'
+        directory to run fitting in
+    remote_info: dict or wfl.autoparallelize.utils.RemoteInfo, or '_IGNORE' or None
+        If present and not None and not '_IGNORE', RemoteInfo or dict with kwargs for RemoteInfo
+        constructor which triggers running job in separately queued job on remote machine.  If None,
+        will try to use env var WFL_EXPYRE_INFO used (see below). '_IGNORE' is for
+        internal use, to ensure that remotely running job does not itself attempt to spawn another
+        remotely running job.
+    verbose: bool default True
+        verbose output
+    do_fit: bool, default True
+        carry out the fit, otherwise only print fitting command 
+    wait_for_results: bool, default True
+        wait for results of remotely executed job, otherwise return after starting job
+    remote_label: str, default None
+        label to match in WFL_EXPYRE_INFO
+    skip_if_present: bool, default False
+        skip if final GAP file exists in expected place
+    
+    """
+    run_dir = Path(run_dir)
+    
+    # If fitting_configs is not given as a file but a memory, 
+	# It should be first written as xyz file. 
+    if not fitting_configs.one_file():
+        fitting_configs = prepare_configs(fitting_configs)
+        # Not so certain how mace_file_base should be set
+        # In iterative training, it should be iteration index.
+        mace_file_base = "test"
+        _write_fitting_configs(fitting_configs, mace_fit_params, mace_file_base)
+
+
+    if skip_if_present:
+        try:
+            print(f"check whether already fitted model exists as {run_dir}/{mace_name}.model")
+            if not Path(f"{run_dir}/{mace_name}.model").is_file():
+                raise FileNotFoundError
+    
+            return mace_name
+        except (FileNotFoundError, RuntimeError):
+            pass
+    
+    
+    if remote_info != '_IGNORE':
+        remote_info = get_remote_info(remote_info, remote_label)
+    
+    
+    if remote_info is not None and remote_info != '_IGNORE':
+        input_files = remote_info.input_files.copy()
+        output_files = remote_info.output_files.copy() + [str(run_dir)]
+    
+    
+        # set number of threads in queued job, only if user hasn't set them
+        if not any([var.split('=')[0] == 'WFL_GAP_FIT_OMP_NUM_THREADS' for var in remote_info.env_vars]):
```

Comment:
If MACE fitting actually uses OpenMP, these should be changed to `WFL_MACE_FIT_OMP_NUM_THREADS`, or if not, all this stuff should be removed.

Reply:
In my simple test, when used with CPU, MACE seems to use multiple cores according to `OMP_NUM_THREADS`. so I retained that part with modified variable name.